ddocs/medic-client/views/data_records_read_by_type/map.js

```diff
@@ -1,34 +1,13 @@
 function(doc) {
```

Comment:
Right. We used to emit the district all the time so district managers could get just data on their district but when we started doing filtered replication this was no longer necessary. I don't know how this view slipped under the radar.